charts/webapp/values.yaml

```diff
@@ -9,7 +9,7 @@ resources:
     cpu: "1"
 image:
   repository: quay.io/wire/webapp
-  tag: "2021-03-15-production.0-e2b28a-v0.28.3-production"
+  tag: 2021-04-01-production.0-254d51-v0.28.3-production
```

Comment:
Oh, are we good without double quotes here? https://github.com/wireapp/wire-server/pull/1477